internal/pkg/model/orchestrator.go

```diff
@@ -33,9 +40,65 @@ func (v *Orchestration) Clone() *Orchestration {
 		return nil
 	}
 	clone := *v
+	clone.Phases = make([]*Phase, 0)
```

Comment:
Upravil som to na `pointre`, ale zabudol som fixnut klonovanie, ... v teste to nebolo pokryte.